src/js/components/homepage/visualizations/treemap/BudgetSubfunctions/BudgetSubfunctions.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+* BudgetSubfunctions.jsx
+* Created by Emily Gullo 05/15/2017
+**/
+
+import React from 'react';
+import _ from 'lodash';
+
+import BudgetSubfunctionsNavigation from './BudgetSubfunctionsNavigation';
+import BudgetSubfunctionsDescription from './BudgetSubfunctionsDescription';
+import BudgetSubfunctionsMap from './BudgetSubfunctionsMap';
+
+const propTypes = {
+    alternateColors: React.PropTypes.array,
+    categories: React.PropTypes.object,
+    colors: React.PropTypes.array,
+    descriptions: React.PropTypes.array,
+    selected: React.PropTypes.number,
+    subfunctions: React.PropTypes.object,
+    toggleSubfunction: React.PropTypes.func,
+    changeActiveSubfunction: React.PropTypes.func,
+    tooltipStyles: React.PropTypes.object,
+    totalNumber: React.PropTypes.number
+};
+
+export default class BudgetSubfunctions extends React.Component {
+    render() {
+        const category = _.find(this.props.categories.children, { id: this.props.selected });
```

Comment:
These three functions are probably better stored in state, since any change to them will trigger a re-render of all child components. By putting them in state, we can more closely control when they change (aka only when the `selected` prop updates).